update-affiliated/update_astropy_helpers.py

```diff
@@ -67,8 +72,7 @@ def open_pull_request(fork, repo):
     os.chdir(tmpdir)
 
     # Clone the repository
-    run_command('git clone {0}'.format(fork.ssh_url))
-    os.chdir(repo.name)
+    run_command('git clone {0} .'.format(fork.ssh_url))
```

Comment:
Note: I had problem `chdir` into code when repo name is different than package name (e.g., `synphot_refactor` vs `synphot`). Not sure how you made it work before, but this change works for me.